hack/docker-push.sh

```diff
@@ -47,37 +47,42 @@ function highest_release() {
     done
 }
 
+triggeredBy=$(echo $GITHUB_REF | cut -d / -f 2)
+if [[ "$triggeredBy" == "heads" ]]; then
+    BRANCH=$(echo $GITHUB_REF | cut -d / -f 3)
+    TAG=
+elif [[ "$triggeredBy" == "tags" ]]; then
+    BRANCH=
+    TAG=$(echo $GITHUB_REF | cut -d / -f 3)
+fi
+
 if [[ "$BRANCH" == "master" ]]; then
     VERSION="$BRANCH"
-elif [[ ! -z "$TRAVIS_TAG" ]]; then
+elif [[ ! -z "$TAG" ]]; then
     # Tags aren't fetched by Travis on checkout, and we don't need them for master
```

Comment:
Might want to remove the reference to Travis in this comment.

Reply:
done!